src/Web/DOM/Element.purs

```diff
@@ -52,6 +53,8 @@ import Web.DOM.Internal.Types (Element) as Exports
 import Web.DOM.Internal.Types (Element, HTMLCollection, Node)
 import Web.DOM.NonDocumentTypeChildNode (NonDocumentTypeChildNode)
 import Web.DOM.ParentNode (ParentNode)
+import Web.DOM.ParentNode (QuerySelector) as Exports
```

Comment:
```suggestion
```

Reply:
Don't I still want to export it for convenience? (Sorry, I don't think I understand)

Reply:
It’s also re-exported on the next line — this is the source of the duplicate error.